script.js

```diff
@@ -1,4 +1,81 @@
+// Create game states
+var playerToRollDice = "rollDice";
+var playerChooseDiceOrder = "chooseDiceOrder";
+
+var player2ToRollDice = "rollDice2";
+var player2ChooseDiceOrder = "chooseDiceOrder2";
+
+var compareScores = "compareScores";
+
+// Default game state
+var currentGameMode = playerToRollDice;
+
+var player1DiceRoll1, player1DiceRoll2;
+var player2DiceRoll1, player2DiceRoll2;
```

Comment:
I'd probably explore the usage of an array here. Although this works, it gets harder to scale the game IF you decide to make a game with multiple dice like say, 5 dice, 6 dice.

you'd have to have the same number of variables with the dice. and it's not super ideal.